### DISCUSSION TITLE
1.6.1-beta1

### DISCUSSION THREAD

**Discussion by chucker** [chucker/Mastonaut#104] [Announcements]
This fixes #103 and tries to mitigate #63 and #95.

<hr /><em>This discussion was created from the release <a href='https://github.com/chucker/Mastonaut/releases/tag/app-1.6.1-beta1'>1.6.1-beta1</a>.</em>

**Comment by akibasocial** [comment]
Fixes #103, can't reproduce the crash on #63 
macOS 12.6.4
